smarts/core/external_provider.py

```diff
@@ -52,7 +52,10 @@ def state_update(
 
     @property
     def action_spaces(self) -> Set[ActionSpaceType]:
+        # What should be here?
+        # pytype: disable=bad-return-type
         return {}
+        # pytype: enable=bad-return-type
```

Comment:
Ah, it confused me for a second but this is a dictionary playing pretend.

```suggestion
        return set()
```